8.observability/README.md

```diff
@@ -0,0 +1,126 @@
+# Observability with Dapr
+
+This sample explores the [observability](https://github.com/dapr/docs/blob/master/concepts/observability/README.md) capabilities of Dapr. Observability includes metric collection, tracing, logging and health checks. In this sample you'll be enabling [distributed tracing](https://github.com/dapr/docs/tree/master/concepts/observability/traces.md) on an application without changing any application code or creating a dependency on any specific tracing system. Since Dapr uses [OpenTelemetry](https://opentelemetry.io/), a variety of observability tools can be used to view and capture the traces.  In this sampple you'll be using [Zipkin](https://zipkin.io/).
+
+In this sample you will;
+
+- Deploy Zipkin and configure it as a tracing provider for Dapr in Kubernetes.
+- Instrument an application for tracing and then deploy it.
+- Troubleshoot a performance issue.
+
+## Prerequisites
+
+This sample builds on the [distributed calculator](../3.distributed-calculator/README.md) sample and requires Dapr to be installed on a Kubernetes cluster along with state store. It is suggested to go through the distributed calculator sample before this one, which sets you up ready for this sample. If you have not done this then;
+
+1. Clone this repo using `git clone https://github.com/dapr/samples.git`
+
+2. [Install Dapr on Kubernetes](https://github.com/dapr/docs/blob/master/getting-started/environment-setup.md#installing-dapr-on-a-kubernetes-cluster).
+3. [Configure Redis](https://github.com/dapr/docs/tree/master/howto/configure-redis) as a state store for Dapr.
+
+## Configure Dapr tracing in the cluster
+
+Review the Dapr configuration file *./deploy/tracing.yaml* below:
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Configuration
+metadata:
+  name: appconfig
+spec:
+  tracing:
+    enabled: true
+    expandParams: true
+    includeBody: true
+```
+
+This configuration file enables Dapr tracing. Deploy the configuration by running:
+
+`kubectl apply -f ./deploy/tracing.yaml`
+
+## Deploy Zipkin to the cluster and set it as the tracing provider
+
+In this sample Zipkin is used for tracing. Examine [*./deploy/zipkin.yaml*](./deploy/zipkin.yaml) and see how it includes three sections:
+
+1. A **Deployment** for Zipkin using the *openzipkin/zipkin* docker image.
+2. A **Service** which will expose Zipkin internally as a ClusterIP in Kubernetes.
+3. A **Component** that defines Zipkin as the tracing provider for Dapr.
+
+Deploy Zipkin to your cluster by running:
+
+`kubectl apply -f ./deploy/zipkin.yaml`
+
+Now that Zipkin is deployed, you can access the Zipkin UI by creating a tunnel to the internal Zipkin service you just created by running:
+
+`kubectl port-forward svc/zipkin 9411:9411`
+
+On your browser go to [http://localhost:9411](http://localhost:9411). You should be able to see the Zipkin dashboard.
+
```

Comment:
I would also use this opportunity to show that tracing is enabled on the cluster by using the dapr CLI. I would add sentence
" You can see that tracing has been enable of the cluster by running the dapr configurations command with the kubernetes option"

Like this

C:\WINDOWS\system32>dapr configurations --kubernetes
  NAME     TRACING-ENABLED  MTLS-ENABLED  MTLS-WORKLOAD-TTL  MTLS-CLOCK-SKEW  AGE  CREATED
  default  false            true          24h                15m              4d   2020-04-01 16:38.04

Reply:
Great suggestion. Added this right after appconfig is applied to the cluster. See  https://github.com/dapr/samples/pull/183/commits/670cd71d452c98433367297fa5e6948cb6ac6c6f. Thank you!